resources/webroot/static/data/de.json

```diff
@@ -3219,8 +3284,10 @@
 		"quest_28": "Nehme %{amount} Schnappschuss/Schnappschüsse",
 		"quest_29": "Kämpfe in %{amount} Team Rocket Kampf/Kämpfe",
 		"quest_30": "Erlöse %{amount} Pokemon",
-        "quest_31": "Find %{amount} Team Rocket Invasion(en)",
-        "quest_32": "%{amount} mal erster Grunt des Tages",
+        	"quest_31": "Find %{amount} Team Rocket Invasion(en)",
+        	"quest_32": "%{amount} mal erster Grunt des Tages",
+		"quest_33": "Unbekannt",
+        	"quest_34": "Erhalte 5 Herzen mit deinem Buddy",
```

Comment:
shouldn't this be %{amount} instead of 5?

Reply:
yes, now it's correct